openghg/store/storage/_localzarrstore.py

```diff
@@ -259,7 +259,7 @@ def delete_all(self) -> None:
         """
         self._check_writable()
         self._stores.clear()
-        shutil.rmtree(self._stores_path, ignore_errors=True)
+        shutil.rmtree(self._stores_path.parent, ignore_errors=True)
```

Comment:
This may be overkill but do you think it may be worth adding a check that `self._stores_path.parent != self._bucket` or something? I can't see a situation where it would be needed, I'm just always a bit nervous about deleting directories especially when they're not direct but derived relative to another directory.